actiontest/readme.md

```diff
@@ -0,0 +1,12 @@
+To run locally. 
+
+1. Install Docker Desktop https://www.docker.com/products/docker-desktop
+2. Install ACT - https://github.com/nektos/act
+3. RUN: act -P ubuntu-latest=nektos/act-environments-ubuntu:18.04 -j test --secret-file actiontest/.secrets --env-file actiontest/.environment
```

Comment:
```suggestion
1. Install Docker Desktop https://www.docker.com/products/docker-desktop
1. Install ACT - https://github.com/nektos/act
1. Copy .secrets-example to .secrets and replace `xxxx` with real values
1. Run `npm run act`
```

Adds the secrets step and prefer the act script. Also switched to let markdown rendering do the numbering correctly, which is a personal preference, because I find it makes it easier to edit later.